front/pages/w/[wId]/members/index.tsx

```diff
@@ -219,7 +211,7 @@ export default function WorkspaceAdmin({
         />
         <ChangeMemberModal
           showModal={changeRoleModalOpen}
-          member={changeRoleMember}
+          member={members.find((m) => m.id === changeRoleMemberId) || null}
```

Comment:
If we really need only the id in the state, maybe we can find only if changeRoleMemberId is not null?